libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/milestone/VoiceInstructionMilestone.java

```diff
@@ -61,13 +62,27 @@ private void clearInstructionList() {
   }
 
   /**
-   * @param previousRouteProgress most recent progress before the current progress
-   * @param routeProgress         the current route progress
-   * @return true if on a new step, false if not
+   * Looks to see if we have a new step.
+   *
+   * @param routeProgress provides updated step information
+   * @return true if new step, false if not
+   */
+  private boolean newStep(RouteProgress routeProgress) {
```

Comment:
Why not add this to RouteUtils?

Reply:
@cammace both `newRoute`and `newStep` require a class field `LegStep` or `DirectionsRoute` that we wouldn't be able to cache in the Util class because it's static right?